requirements/common.txt

```diff
@@ -109,3 +109,10 @@ httplib2==0.9.1
 # Required by django.contrib.migrations
 # sha256: Z4xsNspLAUBRd9qLhO7PkuySyfbHYjlsllu10wXyD4E
 sqlparse==0.1.16
+
+# Required by hawkrest
+# sha256: SNZPN8Hy9utro7Xy2LDByQ4offLtH9-Ah0_tDDYJ61I
+mohawk==0.3.0
```

Comment:
mohawk requires `six`, which is currently only in dev.txt, so needs moving over to common.txt or this will break on prod:
https://github.com/kumar303/mohawk/blob/0.3.0/setup.py#L25